newIDE/app/src/BehaviorsEditor/Editors/BehaviorPropertiesEditor.js

```diff
@@ -24,15 +33,62 @@ const BehaviorPropertiesEditor = ({
     setShouldShowDeprecatedProperties,
   ] = React.useState<boolean>(false);
 
-  const propertiesSchema = React.useMemo(
+  const basicPropertiesSchema = React.useMemo(
     () =>
       propertiesMapToSchema(
         behavior.getProperties(),
         behavior => behavior.getProperties(),
         (behavior, name, value) => {
           behavior.updateProperty(name, value);
         },
-        object
+        object,
+        'Basic'
+      ),
+    [behavior, object]
+  );
+
+  const areAdvancedPropertiesModified = React.useCallback(
+    () => {
+      const behaviorMetadata = gd.MetadataProvider.getBehaviorMetadata(
+        gd.JsPlatform.get(),
+        behavior.getTypeName()
+      );
+      if (gd.MetadataProvider.isBadBehaviorMetadata(behaviorMetadata)) {
+      }
```

Comment:
to finish?

Reply:
I removed it. Bad metadata have empty properties so it already returns false.